app/views/stats_graph.php

```diff
@@ -0,0 +1,41 @@
+<div id="graphdiv"></div>
+
+<script src="https://l10n.mozilla-community.org/~pascalc/web_l10n_stats/dygraph-combined.js"></script>
```

Comment:
Yes, I don't want to have to update the library in two locations and since we are fetching the library on the same domain name, we don't have to worry about it not being available.